lib/src/command/lish.dart

```diff
@@ -61,7 +61,9 @@ class LishCommand extends PubCommand {
         negatable: false,
         help: 'Publish without confirmation if there are no errors.');
     argParser.addOption('server',
-        help: 'The package server to which to upload this package.');
+        help: 'The package server to which to upload this package.\n'
+            'DEPRECATED: use `publish_to` in your pubspec.yaml or set the \n'
+            '\$PUB_HOSTED_URL environment variable.');
```

Comment:
Similar here, can't we just hide the flag?

Reply:
Done